src/main/java/org/icgc/argo/workflow_management/graphql/GraphQLProvider.java

```diff
@@ -66,6 +63,12 @@ private RuntimeWiring buildWiring() {
     return RuntimeWiring.newRuntimeWiring()
         .scalar(ExtendedScalars.Json)
         .type(newTypeWiring("Mutation").dataFetchers(mutationDataFetcher.mutationResolvers()))
+        .type(newTypeWiring("Query").dataFetcher("activeRuns", activeRunsDataFetcher))
+        // Field visibility hides from schema, but they are executable
```

Comment:
I see this comment now, yeah maybe just add this to the class as well

Reply:
especially as we may want to use that class again but this particular line may get removed